src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -61,6 +67,8 @@
 ST_FAILED = "FAILED"
 ST_CANCELED = "CANCELED"
 
+# list of feature status that are considered as disabled (not to be acquired)
+DISABLED_FEATURE_STATUS = [FEATURE_DEACTIVE]
```

Comment:
I don't get why this is a list

Reply:
wasn't sure if I would dynamically add status' to that during the workflow, but ill remove it now